textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/keywordmatch/KeywordMatcherTest.java

```diff
@@ -175,15 +126,15 @@ public void testKeywordMatcher() throws Exception {
     @Test
     public void testSingleWordQueryInStringField() throws Exception {
         //Prepare Query
-        String query = "Bruce";
+        String query = "bruce";
```

Comment:
@Test
    public void testSingleWordQueryInStringField() throws Exception {
        //Prepare Query
        String query = "BRUCE";
        ArrayList<Attribute> attributeList = new ArrayList<>();
        attributeList.add(TestConstants.FIRST_NAME_ATTR);
        attributeList.add(TestConstants.LAST_NAME_ATTR);
        attributeList.add(TestConstants.DESCRIPTION_ATTR);

        //Prepare expected result list
        List<Span> list = new ArrayList<>();
        Span span = new Span("firstName", 0, 5, "BRUCE", "bruce");
        list.add(span);
        Attribute[] schemaAttributes = new Attribute[TestConstants.ATTRIBUTES_PEOPLE.length + 1];
        for(int count = 0; count < schemaAttributes.length - 1; count++) {
            schemaAttributes[count] = TestConstants.ATTRIBUTES_PEOPLE[count];
        }
        schemaAttributes[schemaAttributes.length - 1] = SchemaConstants.SPAN_LIST_ATTRIBUTE;

        IField[] fields1 = { new StringField("bruce"), new StringField("john Lee"), new IntegerField(46),
                new DoubleField(5.50), new DateField(new SimpleDateFormat("MM-dd-yyyy").parse("01-14-1970")),
                new TextField("Tall Angry"), new ListField<>(list) };
        ITuple tuple1 = new DataTuple(new Schema(schemaAttributes), fields1);
        List<ITuple> expectedResultList = new ArrayList<>();
        expectedResultList.add(tuple1);

        //Perform Query
        List<ITuple> resultList = getPeopleQueryResults(query, attributeList);

        //Perform Check
        boolean contains = TestUtils.containsAllResults(expectedResultList, resultList);
        Assert.assertTrue(contains);
        Assert.assertEquals(1,resultList.size());
    }


I made changes to your test case and it's failing. As @chenlica  pointed out, our matching is case insensitive shouldn't this pass? Am I missing something?